src/software/ai/hl/stp/play/enemy_ball_placement_play_test.py

```diff
@@ -11,7 +12,7 @@
 from proto.ssl_gc_common_pb2 import Team
 
 
-def test_two_ai_ball_placement(simulated_test_runner):
+def test_two_ai_ball_placement_in_stadium(simulated_test_runner):
```

Comment:
let's parametrize the two tests with run_enemy_ai as well (so the enemy AI is running HaltPlay)

Reply:
I don't quite understand this one. The run_enemy_ai parameter will determine whether the ai will be run. I think it is set to run right now already, are we possibly running this test without the enemy robots moving?

Reply:
yes exactly that's what I mean. Let's have a version of the tests you made with a static enemy AI

Reply:
bump